static/app/components/events/autofix/autofixChanges.tsx

```diff
@@ -242,14 +225,17 @@ export function AutofixChanges({
   }
 
   return (
-    <Content>
-      {step.changes.map((change, i) => (
-        <Fragment key={change.repo_external_id}>
-          {i > 0 && <Separator />}
-          <AutofixRepoChange change={change} groupId={groupId} isLastStep={isLastStep} />
-        </Fragment>
-      ))}
-    </Content>
+    <ChangesContainer>
+      <ClippedBox clipHeight={408}>
+        <h6>Fixes</h6>
```

Comment:
Oh and I'd rather not use `h6` here. It's good to have a header hierarchy generally, but having a single h6 without h2, h3, etc will confuse things. I'd rather just use a styled div to get the look you want.